sendgrid/helpers/inbound/config.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, **opts):
             'path', os.path.abspath(os.path.dirname(__file__))
         )
         with open('{0}/config.yml'.format(self.path)) as stream:
-            config = yaml.load(stream)
+            config = yaml.load(stream, Loader=yaml.FullLoader)
```

Comment:
Why is this needed?

Reply:
Deprecation of the default Loader: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation